theorems/T000205.md

```diff
@@ -0,0 +1,12 @@
+---
+uid: T000205
+if:
+  P000172: true
+then:
+  P000173: true
+refs:
+- mr: MR2049453
+  name: Encyclopedia of general topology
+---
+
+Follows directly from the definitions.
```

Comment:
I find the comparable relationship between FU and sequential to be subtle. It's good to get this merged in but I'll make a separate PR proposing a bit more detail.